### ISSUE TITLE
Error to changing an event graphic using event pages (missing and wrong graphics)

### ISSUE THREAD

**Issue by Br4ssman** [EasyRPG/Player#1587]
**Tested in**: Player 0.5.4 (continuous).
**How to reproduce it**: Create an event with several pages and graphics (fixed graphic). A parallel process event will change among pages using variables and will make "Set Move Route".
**Platform**: Windows 7 (x64)

**Test case**:
[Changing charset graphic via event pages](https://github.com/EasyRPG/Player/files/2820062/TestCase.-.Change.events.graphic.zip)

**Comment by Ghabry** [comment]
Event trace for this:

```
# Session start at 2018-12-19 11:35:56
# F	frame
# E	evt_id	page_id	line_id
# M	evt_id	line_id
F	0
E	2	1	0
E	2	1	1
E	2	1	2
F	36
M	1	0
M	1	1
F	44
M	1	2
F	52
M	1	3
F	60
M	1	4
E	2	1	3
E	2	1	4
F	78
E	2	1	5
E	2	1	6
E	2	1	7
F	79
M	1	0
M	1	1
F	87
M	1	2
F	95
M	1	3
F	103
M	1	4
E	2	1	8
F	104
E	2	1	0
E	2	1	1
E	2	1	2
F	105
M	1	0
M	1	1
F	113
M	1	2
F	121
M	1	3
F	129
M	1	4
E	2	1	3
E	2	1	4
F	147
E	2	1	5
E	2	1	6
E	2	1	7
F	148
M	1	0
M	1	1
F	156
M	1	2
# Session end at 2018-12-19 11:36:01
```

**Comment by fmatthew5876** [comment]
This looks to be fixed in #1601 

At least that the character animation looks the same.
